includes/base/widget-base.php

```diff
@@ -344,7 +344,7 @@ protected function get_initial_config() {
 		$config = [
 			'widget_type' => $this->get_name(),
 			'keywords' => $this->get_keywords(),
-			'categories' => $this->get_categories(),
+			'categories' => apply_filters( 'elementor/widget/categories', $this->get_categories(), $this ),
```

Comment:
See https://github.com/elementor/elementor/pull/13635/files?file-filters%5B%5D=.php#diff-7801e0142694c4ab4d9c97cb34b933f1f735d177ea8951c272e01c78cd682c63R68

Reply:
@matipojo The `elementor/document/config` filter represents additional settings, and is eventually merged using `array_replace_recursive`, which means - when I add the `favorites` category, the others disappeared. Therefore, I think the current approach is more suitable.

Reply:
How it works in there?